.github/workflows/ci.yml

```diff
@@ -1,17 +1,58 @@
-name: GitHub Actions Demo
-on: [push, pull_request]
+# This workflow uses actions that are not certified by GitHub.  They are
```

Comment:
what's not certified by github here?  This is an area I'm not familiar with, but none of this looks like it's using 3rd party services.  What am I missing>

Reply:
The GH documentation points to a repo with boilerplate recipes, where I took this from. I think that's what they mean, use this boilerplate at your own peryl :P

Reply:
hehehe, okee dokee.